### PR TITLE
[Validator] Allow skipping import validation

### PR DESCRIPTION
**Pull request by segiddins**
- [ ] Specs
- [ ] CHANGELOG
- [ ] Update cocoapods-trunk

### CODE REVIEW

File: lib/cocoapods/validator.rb [hunk 1]
```diff
@@ -424,7 +428,7 @@ def write_app_import_source_file(pod_target)
           f << "@import Foundation;\n"
           f << "@import UIKit;\n" if consumer.platform_name == :ios || consumer.platform_name == :tvos
           f << "@import Cocoa;\n" if consumer.platform_name == :osx
-          f << "#{import_statement}int main() {}\n"
+          f << import_statement
```
Comment (segiddins) [inline lib/cocoapods/validator.rb]: revert this change

**Comment by igagis** [comment]
Hi,
any progress on this?

**Comment by segiddins** [comment]
@igagis it's still awaiting someone writing tests

**Comment by segiddins** [comment]
Closing in favor of https://github.com/CocoaPods/CocoaPods/issues/6420
